provision/ec2.go

```diff
@@ -57,6 +58,14 @@ func NewEC2ProvisionerFromProfile(user, profile, path string, out ...io.Writer)
 // GetUser returns the user attached to given credentials
 func (p *EC2Provisioner) GetUser() string { return p.user }
 
+// GetHomePath returns the homepath based on the operating system used
+func GetHomePath() string {
```

Comment:
oh and we might want to move this function to package `local` instead, since it pertains to local filesystem and might come in handy in the future

https://github.com/ubclaunchpad/inertia/tree/master/local

Reply:
Yeah good point. I was wondering if go had something for that. Would definitely help with the home paths in other windows bug fixes.

Reply:
HomePath function has been migrated to local folder. Need the commit merged before I can import the function in the EC2 homepath fetching.